shiny/ViewMet/server.R

```diff
@@ -141,7 +141,7 @@ server <- function(input, output, session) {
       
       site = query.site(con = bety$con, siteid)
       
-      vars_in_file <- ncdf4::nc_open(rv$load.paths[i]) %>% ncdf4.helpers::nc.get.variable.list()
+      vars_in_file <- ncdf4::nc_open(rv$load.paths[i]) %>% names(nc[[]])
```

Comment:
This is a pre-existing issue, but one we should fix while we're at it: By `nc_open`ing a file inside a pipeline with no matching `nc_close`, we leak the filehandle.

```suggestion
      current_nc <- ncdf4::nc_open(rv$load.paths[i])
      vars_in_file <- names(current_nc)
      ncdf4::nc_close(current_nc)
```

cc @bcow does this look right?

Reply:
@infotroph you mean,
`vars_in_file <- names(current_nc[["var"]])` ? :smiley:

Reply:
Yes, I do. Thanks for catching that

Reply:
@bcow I have made these changes in the last commit, please let me know if it's not okay.